src/js/components/Form/stories/Controlled.js

```diff
@@ -29,20 +29,24 @@ const defaultValue = {
 
 const Example = () => {
   const [value, setValue] = React.useState(defaultValue);
+  const onChange = React.useCallback(nextValue => {
```

Comment:
Reminder to look at whether the hook `useCallback` is still needed.

Reply:
removed `useCallback()` in stories, no longer needed. #